src/systems/thruster/Thruster.cc

```diff
@@ -347,6 +374,9 @@ void Thruster::Configure(
 
     feedbackTopic = gz::transport::TopicUtils::AsValidTopic(
       "/model/" + ns + "/joint/" + jointName + "/ang_vel");
+    
+    this->dataPtr->deadbandTopic = gz::transport::TopicUtils::AsValidTopic(
```

Comment:
check if the output is `empty()` and handle the error

Reply:
Yes. That'd be a good idea.